### ISSUE TITLE
Elm files should have a dedicated icon instead of the default text icon

### ISSUE THREAD

**Issue by aminnairi** [ryanoasis/vim-devicons#288]
#### Required Info

- [x] I have searched the [issues](https://github.com/ryanoasis/vim-devicons/issues) for my issue and found nothing related and/or helpful
- [x] I have read or at least glanced at the [FAQ](https://github.com/ryanoasis/vim-devicons/wiki/FAQ-&-Troubleshooting)
- [x] I have read or at least glanced at the [Wiki](https://github.com/ryanoasis/vim-devicons/wiki)
- [x] `VimDevIcons` is the last loaded vim plugin?
- [x] What **version** of vim are you using?

```console
$ vim --version | head -3
VIM - Vi IMproved 8.1 (2018 May 18, compiled Jul 29 2019 20:38:53)
Included patches: 1-1776
Compiled by Arch Linux
```

- [x] What **version** of vim-devicons are you using?

```
0.11.0
```

- [x] Are you using vim from the terminal or a GUI vim?

```
Terminal
```

- [x] Are you using Mac, Linux or Windows?

```console
$ uname -a
Linux archlinux 4.19.66-1-lts #1 SMP Sat Aug 10 13:13:28 CEST 2019 x86_64 GNU/Linux
```

#### Optional Info

- [x] Provide **screenshots** where appropriate

![image](https://user-images.githubusercontent.com/18418459/66274691-8ae1f480-e881-11e9-9b25-39117c5817dc.png)

I don't know how vim-devicons works under the hood (if it is using images, SVGs or anything else) but here is the logo for the Elm programming language:

![image](https://user-images.githubusercontent.com/18418459/66274778-591d5d80-e882-11e9-92a8-57ca59c9de05.png)

And [here](https://elm-lang.org/) is the documentation for the Elm programming language just for the sake of curiosity.

**Comment by ryanoasis** [comment]
thanks, well we could make use of `e62c` (already in Nerd Fonts)

**Comment by kazukazuinaina** [comment]
@aminnairi 

I wrote `.elm`'s patch

**Comment by ryanoasis** [comment]
closed via #289
